Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -89,10 +87,7 @@ extension AlertManager: AlertManagerResponder {
         if let responder = responders[identifier.managerIdentifier]?.value {
             responder.acknowledgeAlert(alertIdentifier: identifier.alertIdentifier)
         }
-        // Also clear the alert from the NotificationCenter
-        log.debug("Removing notification %@ from delivered & pending notifications", identifier.value)
-        userNotificationCenter.removeDeliveredNotifications(withIdentifiers: [identifier.value])
-        userNotificationCenter.removePendingNotificationRequests(withIdentifiers: [identifier.value])
```

Comment:
This line "removePendingNotificationRequests" was removed, but only after being moved below...